src/components/editor/faultTree/Editor.tsx

```diff
@@ -190,35 +195,55 @@ const Editor = ({ setAppBarName }: DashboardTitleProps) => {
     const result = list.reduce((maxItem, currentItem) => {
       return currentItem.probability > maxItem.probability ? currentItem : maxItem;
     });
-    return result
+    return result;
   };
 
   const handleOnScenarioSelect = (scenario: FaultEventScenario) => {
     setFaultEventScenarios([scenario]);
     refreshTree();
   };
 
+  const redirectToInstance = (navigateFrom: string) => {
+    if (faultTree?.manifestingEvent?.children) {
+      let map: { [name: string]: string } = {};
+      const children = faultTree?.manifestingEvent?.children;
+      for (let i = 0; i < children.length; i++) {
+        const node = children[i];
+        const nodeKey = node.iri;
+        const nodeValue = node?.references?.isPartOf;
+        if (nodeKey && nodeValue) {
+          map[nodeKey] = nodeValue;
+        }
+      }
+      const redirectTo = map[navigateFrom].split("/").pop();
+      history(`/fta/${redirectTo}`);
+    }
+  };
+
   return (
     <div onContextMenu={handleOnContextMenu}>
-      <EditorCanvas
-        treeName={faultTree?.name}
-        rootEvent={rootEvent}
-        onEventUpdated={handleEventUpdate}
-        sidebarSelectedEvent={sidebarSelectedEvent}
-        onElementContextMenu={handleContextMenu}
-        onElementPointerClick={handleElementPointerClick}
-        onBlankPointerClick={handleBlankPointerClick}
-        onCutSetAnalysis={handleCutSetAnalysis}
-        onConvertToTable={() => setFailureModesTableOpen(true)}
-        onNodeMove={handleMoveEvent}
-        setHighlightedElement={setHighlightedElementView}
-        refreshTree={refreshTree}
-        faultEventScenarios={faultEventScenarios}
-        possibleFaultEventScenarios={faultTree?.faultEventScenarios ? faultTree?.faultEventScenarios : []}
-        showPath={showPath}
-        showTable={showTable}
-        onScenarioSelect={(scenario: FaultEventScenario) => handleOnScenarioSelect(scenario)}
-      />
+      {faultTree && (
+        <EditorCanvas
+          treeName={faultTree?.name}
```

Comment:
IMO it can be replaced by `{faultree.name}` for readability as it is already safeguarded by `faultTree &&`, but it is just a personal opinion, no need to change it :)